Formula/s/seclists.rb

```diff
@@ -0,0 +1,26 @@
+class Seclists < Formula
+  desc "Collection of multiple types of lists used during security assessments"
+  homepage "https://github.com/danielmiessler/SecLists"
+  url "https://github.com/danielmiessler/SecLists/tree/2024.1"
+  sha256 "da0a9314cde32c67622c8030e846f94461e6c2ccaff5f529ee066e898097bd82"
+  license "MIT"
+
+  def install
+    mkdir "#{prefix}/seclists"
+    share.install_symlink "#{prefix}/seclists"
+    ohai "Size of a complete clone of SecLists is currently at 1.2 GB"
+    ohai "Cloning this repository should take 4-5 minutes at 5MB/s speeds."
+    system "git", "clone", "git@github.com:danielmiessler/SecLists.git", "#{prefix}/seclists"
+  end
+
+  test do
+    assert_predicate prefix/"seclists", :exist?
+    # because it's only list files in the repo, the only valid test is to
```

Comment:
Then I don't understand what benefit there is to having a Homebrew package. If there is nothing to compile, why not tell people to `git clone` the repo?

Reply:
This repo is also a package of apt and pacman, because I started using it on a MacOS system also I thought I would be great to have it in homebrew to keep my packages installation neat and clean. All the packages in homebrew need to compile something ?

Reply:
They don't need to by strict definition, but I don't understand why anyone would think to install it through brew if you can get exactly the same with git clone.

Reply:
It's really about lazyness, it's much easier to install things this way then to have to go to github get the url of the repo, download it, unarchive everything etc... There's not much more to it.